Content/Library/tests/MyLib.1.Tests/MyLib.1.Tests.fsproj

```diff
@@ -2,7 +2,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
     <PropertyGroup>
         <OutputType>Exe</OutputType>
-        <TargetFrameworks>netcoreapp2.1;netcoreapp2.0;net461</TargetFrameworks>
+        <TargetFrameworks>netcoreapp2.1;netcoreapp2.2;net461</TargetFrameworks>
```

Comment:
This didn't change with this MR, but should a library include a .netstandard2.0 target by default?  I'm thinking a good matrix is
* latest .net framework of a given minor version (ie net47)
* latest .netstandard 
* latest .netcoreapp

whatcha think?

Reply:
So you're thinking off adding a `netcoreappx.x` to the project under src for Library?  Sounds like a good idea since netstandard might lag behind.

I chose` net461` because I've always felt targeting lower is better for libraries.